core/sail/shacl/src/main/java/org/eclipse/rdf4j/sail/shacl/planNodes/ValuesBackedNode.java

```diff
@@ -0,0 +1,118 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Eclipse RDF4J contributors.
```

Comment:
That looks like the wrong year for a new file

Reply:
Will fix